### DISCUSSION TITLE
Everything is very nice, but why V?

### DISCUSSION THREAD

**Discussion by skrtdev** [vlang/v#8372] [General]
I don't wanna know why I should use V instead of another language. I'd like to know why this language is named 'V'. Probably there are some causes

**Comment by medvednikov** [comment]
https://github.com/vlang/v/wiki/FAQ

- Why "V"?

Initially the language had the same name as the product it was created for: Volt. The extension was ".v", I didn't want to mess up git history, so I decided to name it V :)

It's a simple name that reflects the simplicity of the language, and it's easy to pronounce for everyone in the world.

Please note that the name of the language is "V", not "Vlang" or "V-Lang" etc.

The name is not very searchable (like Go), so use #vlang on Twitter, vlang on Google etc.

**Comment by skrtdev** [comment]
Oh ok thank you. Everything is clearer now
